privacy-on-beam/pbeam/distinct_id.go

```diff
@@ -68,32 +72,39 @@ type DistinctPrivacyIDParams struct {
 //
 // DistinctPrivacyID transforms a PrivatePCollection<V> into a
 // PCollection<V,int64>.
-func DistinctPrivacyID(s beam.Scope, pcol PrivatePCollection, params DistinctPrivacyIDParams) beam.PCollection {
-	s = s.Scope("pbeam.DistinctPrivacyID")
+func DistinctPrivacyID(s beam.Scope, pcol PrivatePCollection, params DistinctPrivacyIDParams, partitions ...beam.PCollection) beam.PCollection {
+	if len(partitions) > 1 {
+		log.Exitf("Only one partition PCollection can be specified. %v were specified.", len(partitions))
+	}
 	// Obtain type information from the underlying PCollection<K,V>.
 	idT, partitionT := beam.ValidateKVType(pcol.col)
-
-	var noiseKind noise.Kind
-	if params.NoiseKind == nil {
-		noiseKind = noise.LaplaceNoise
-		log.Infof("No NoiseKind specified, using Laplace Noise by default.")
-	} else {
-		noiseKind = params.NoiseKind.toNoiseKind()
+	if len(partitions) == 1 {
+		if partitionT.Type() != partitions[0].Type().Type() {
```

Comment:
Same as above - can we assume all the elements of the partitions collection are the same type?